examples/timeseries/timeseries_weather_forecasting.py

```diff
@@ -0,0 +1,418 @@
+"""
+Title: Timeseries forecasting for weather prediction
+
+**Authors:**
+
+ - [Prabhanshu Attri](https://prabhanshu.com/github),
+ - [Yashika Sharma](https://github.com/yashika51),
+ - [Kristi Takach](https://github.com/ktakattack),
+ - [Falak Shah](https://github.com/falaktheoptimist)
+<br>
+
+**Date created:** 2020/06/23  <br>
+**Last modified:** 2020/06/30  <br>
+**Description:** This notebook demonstrates how to do timeseries forecasting
+using a LSTM model.
+"""
+
+"""
+## Setup
+
+
+As of 25/6/20, `timeseries_dataset_from_array()` is available with TensorFlow nightly.
+
+If the following cell doesn't work, try uninstalling Tensorflow and Keras using
+this command.
+
+```
+!pip uninstall tf-nightly keras -y
+```
+
+# install tf-nightly for timeseries_dataset_from_array
+!pip install 'tf-nightly==2.3.0.dev20200623'
+!pip install 'keras==2.4.0'
+
+**Note**: Please restart the runtime using the button above in order to load
+installed Tensorflow and Keras versions.
```

Comment:
Please remove this section and simply state: "This example requires TensorFlow 2.3 or higher."

Note that 2.3 will be the default TensorFlow in a couple of weeks. There's already a release candidate available.